pex/tools/commands/virtualenv.py

```diff
@@ -244,13 +244,9 @@ def rewrite_scripts(
         python_args=None,  # type: Optional[str]
     ):
         # type: (...) -> Iterator[str]
-        python_scripts = []
-        for executable in self.iter_executables():
-            if executable in self._base_bin:
-                continue
```

Comment:
To check my understanding...removing this condition is this part of the change?

> Fix venv population collision detection logic to not only consider the destination path, but also the source content.

Reply:
It is. It supports the test here: https://github.com/pantsbuild/pex/pull/1572/files#diff-1f8e2f7c4438642ba2a6b917206e0a5d453d8b8349b525edb421f68a9f774248R102 by ensuring the `pex` script, which is included in _base_bin due to how tox works, is re-written to have its shebang be something real instead of `#!python` which is what it remained with before this fix. So its not logically part of this change, but it supports this change. Seems too small to break out on its own.